.template/addons/github/.github/workflows/deploy_aws_fargate.yml.tt

```diff
@@ -0,0 +1,50 @@
+name: Deploy to AWS Fargate
+
+on:
+  workflow_run:
+    workflows:
+      - Test
+    branches:
+      - main
```

Comment:
some project prefer to keep using `master`, maybe we can adding `master` along with `main` too, what do you think @longnd

Reply:
I added the master branch support in ab98c46.